cli/impl/src/main/java/io/helidon/build/cli/impl/Helidon.java

```diff
@@ -85,4 +97,39 @@ private static void execute(String[] args, boolean embedded) {
                      .execute()
                      .runExitAction();
     }
+
+    private static String[] preProcessArgs(String[] args) {
```

Comment:
`--args-file` needs to be handled by the hardness as a built-in argument similar to `--help`.
Any CLI based on the harness and any sub-command should support this built-in argument.